### ISSUE TITLE
No config files of any type on Fabric 1.20.1

### ISSUE THREAD

**Issue by ididifdgb** [Cadiboo/NoCubes#124]
Created new instance on Modrinth, Fabric 1.20.1. Using NoCubes latest Fabric Version from August 2024 for 1.20.1. I only have Fabric API and NoCubes. No Config file. No Server config file in my save folder. I also tried with Cloth Config installed, and nothing changed.

**Comment by Cadiboo** [comment]
Yep, no config on fabric yet, will be added in a future release.
